docs/gcp-wif-configuration.md

```
@@ -0,0 +1,280 @@
+- `vercel-provider-preview`
+- `vercel-provider-production`
+
+これにより condition と allowedAudiences の混線を防げる。
+
+## Cloud Run URL の正規化ルール
+
+**重要**: Cloud Run は複数の URL 形式でアクセス可能だが、`targetAudience` として使う URL は統一すること。
+
+### 正規 URL
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)